doc/config.rst

```diff
@@ -1116,10 +1116,21 @@ that use Sphinx's HTMLWriter class.
 
 .. confval:: html_experimental_html5_writer
 
-   Output is processed with HTML5 writer.  This feature needs docutils 0.13 or newer.  Default is ``False``.
+   Output is processed with HTML5 writer.  This feature needs docutils 0.13 or
+   newer.  Default is ``False``.
 
    .. versionadded:: 1.6
 
+.. confval:: html_footnotes_options
+
+   If the configuration value contains ``bottom``, all footnotes are rendered
+   at the bottom of a document.  With the additional ``rubric`` or ``title`` a
+   rubric directive respectively a title are prepended.
+   The text of a footnote will be rendered as tooltip when ``tooltip`` is in
+   the configuration value.
```

Comment:
I'd like to improve `bottom` option to support other builders. Personally, I've usually placed footnotes at the end of each sections. So it would be nice if this also supports it. Are you interested in?

Reply:
well, i don't have the time to implement that now, but i'd change the current option's name to `page-bottom` (or should it be `content-bottom`?) for that extension.